build/site/specifications/1.3/index.html

```diff
@@ -28,13 +28,13 @@
         <div class="navbar-item has-dropdown is-hoverable">
           <div class="navbar-link">Download</div>
           <div class="navbar-dropdown">
-            <a class="navbar-item" href="../1.3/specs/_attachments/C2PA_Specification.pdf">Technical Specifications</a> 
-            <a class="navbar-item" href="../1.2/explainer/_attachments/Explainer.pdf">Explainer</a> 
-            <a class="navbar-item" href="../1.2/guidance/_attachments/Guidance.pdf">Guidance for Implementers</a> 
-            <a class="navbar-item" href="../1.1/ux/_attachments/UX_Recommendations.pdf">User Experience Guidance</a> 
+            <a class="navbar-item" href="../1.0/specs/_attachments/C2PA_Specification.pdf">Technical Specifications</a> 
```

Comment:
Again, are these paths pointing to the right versions? When I open the site directory in a browser, the "Download" drop down at the top points to 1.0 versions of everything.